tf_big/README.md

```diff
@@ -0,0 +1,45 @@
+## TF Big
+
+TF Big implements some basic operations on big integers. TF Big uses [libgmp](https://gmplib.org/) for its optimized big integer routines.
+
+## Requirements
+
+For Linux, the recommended way to build and run tests is using docker. Please these instructions for installing docker on Ubuntu.
+
+libgmp needs to be installed prior to building TF Big. libgmp is provided simply through [Homebrew](https://brew.sh/) and through apt for Ubuntu. Install using one of the following methods:
```

Comment:
Does GMP need to be installed for using TF Big as well?

Reply:
gmp and libgmp as the same thing to me. Maybe we should call it gmp instead of libgmp seems like a more common way of referring to it.

Reply:
Does either need to be installed? Wondering if we should ship a static version of GMP together with TF Big.

Reply:
Hmmm thought we decided somewhere that this needs to be linked dynamically. We can statically link gmp to the shared object though if that's what you mean. I don't think we should be shipping a separate shared object/archive for GMP though. If we want to go the static route I think it can be done in another PR its a little involved and installing gmp seems pretty easy for a user to do right now.

Reply:
reading it again I agree it could be clearer: https://github.com/tf-encrypted/tf-encrypted/issues/568#issuecomment-503054264

bottom line is that we need a dynamic link *somewhere* but think it should be between eg TFE and TFBig, and not between TFBig and GMP. Problem is that I don't think we should expect users to install GMP manually.

Reply:
Alright! Sounds good to me. Mind if I do it in another PR soon after this?

Reply:
sounds good!

Reply:
Added this issue: https://github.com/tf-encrypted/tf-big/issues/6